EnglishFilter/sections/specific.txt

```diff
@@ -706,6 +706,7 @@ nstnews.net##p[style="text-align: center;"] > a[target="_blank"]
 bloomberg.com##.leaderboard-container
 1337x.*##.no-top-radius > .clearfix > ul[class*=" "] > li > a[href^="/rmusic-"]
 1337x.*##.no-top-radius > .clearfix > ul[class*=" "] > li > a[href^="/dirdownl-"]
+fastbikesmag.com###secondary > aside[id^="custom_html-"]
```

Comment:
This rule blocks newletter, change to `fastbikesmag.com##.main-sidebar > aside[id^="custom_html-"]`